public/jsx/components/card-item.jsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var React = require('react');
+var CardIcon = require('components/card-icon');
+
+
+module.exports = React.createClass({
+
+  displayName: 'CardItem',
+
+  propTypes: {
+    id: React.PropTypes.number.isRequired,
+    onChangeHandler: React.PropTypes.func.isRequired,
+    resource_uri: React.PropTypes.string.isRequired,
+    truncated_id: React.PropTypes.string.isRequired,
+    type_name: React.PropTypes.string.isRequired,
+  },
+
+  render: function() {
+    var cardType = this.props.type_name.toLowerCase();
+    var cardText = '●●●● ●●●● ●●●● ' + this.props.truncated_id;
```

Comment:
Yeah, that's a todo - I can't test other types that change patterns yet due to the pattern bug in the lib we're using as I mentioned in the description above.